docs/UserGuide.md

```diff
@@ -83,13 +83,13 @@ For example in `add -n scrambled eggs -p 100`, `add` is a *Command Word* and `-n
 
 **How to intepret the each command's format**:
 
-We will follow the following convention for each command's format:
+You should follow the following convention for each command's format:
```

Comment:
I dont think this one need to change to you should?

Reply:
Alright!